commit ec313b5be0d1a448c4c515f148b290d53c528d9b
Author: q66 <q66@chimera-linux.org>
Date:   Thu Aug 31 20:51:57 2023 +0200

    fix aarch64 build
    
    ../../gdb/aarch64-linux-nat.c:448:19: error: variable-sized object may not be initialized

diff --git a/gdb/aarch64-linux-nat.c b/gdb/aarch64-linux-nat.c
index e415823..bd724e4 100644
--- a/gdb/aarch64-linux-nat.c
+++ b/gdb/aarch64-linux-nat.c
@@ -445,7 +445,8 @@ fetch_tlsregs_from_thread (struct regcache *regcache)
   gdb_assert (regno != -1);
   gdb_assert (tdep->tls_register_count > 0);
 
-  uint64_t tpidrs[tdep->tls_register_count] = { 0 };
+  uint64_t tpidrs[tdep->tls_register_count];
+  memset(tpidrs, 0, sizeof(uint64_t) * tdep->tls_register_count);
   struct iovec iovec;
   iovec.iov_base = tpidrs;
   iovec.iov_len = sizeof (tpidrs);
@@ -471,7 +472,8 @@ store_tlsregs_to_thread (struct regcache *regcache)
   gdb_assert (regno != -1);
   gdb_assert (tdep->tls_register_count > 0);
 
-  uint64_t tpidrs[tdep->tls_register_count] = { 0 };
+  uint64_t tpidrs[tdep->tls_register_count];
+  memset(tpidrs, 0, sizeof(uint64_t) * tdep->tls_register_count);
 
   for (int i = 0; i < tdep->tls_register_count; i++)
     {
